src/commands/publish/route.rs

```diff
@@ -28,103 +34,71 @@ impl Route {
         {
             failure::bail!("You must provide a zone_id in your wrangler.toml before publishing!");
         }
-        let msg_config_error = format!("{} Your project config has an error, check your `wrangler.toml`: `route` must be provided.", emoji::WARN);
+        let msg_config_error = format!(
+            "{} Your project config has an error, check your `wrangler.toml`: `route` must be provided.", 
+            emoji::WARN
+        );
         Ok(Route {
             script: Some(target.name.to_string()),
             pattern: target.route.clone().expect(&msg_config_error),
         })
     }
+}
 
-    pub fn publish(
-        user: &GlobalUser,
-        target: &Target,
-        route: &Route,
-    ) -> Result<(), failure::Error> {
-        if route.exists(user, target)? {
-            return Ok(());
-        }
-        create(user, target, route)
+pub fn publish(user: &GlobalUser, target: &Target, route: &Route) -> Result<(), failure::Error> {
+    if exists(user, target, route)? {
+        return Ok(());
     }
+    create(user, target, route)
+}
 
-    pub fn exists(&self, user: &GlobalUser, target: &Target) -> Result<bool, failure::Error> {
-        let routes = get_routes(user, target)?;
+fn exists(user: &GlobalUser, target: &Target, route: &Route) -> Result<bool, failure::Error> {
```

Comment:
nit: should probably name this `route_exists` or maybe put this on `Route` (maybe out of scope)

Reply:
i took it off of route. i don't think this will stay around very long.

Reply:
renamed route_exists